test/linter.js

```diff
@@ -82,13 +82,16 @@ async function getTagsFromConfig(config) {
 function execLinterCommand(args) {
   var cmd = `npx autorest@2.0.4152 --validation --azure-validator --message-format=json ${args}`.trim();
   console.log(`Executing: ${cmd}`);
+  var errorsFound = false;
   try {
     let result = execSync(cmd, { encoding: 'utf8', maxBuffer: 1024 * 1024 * 64 });
     console.error(result);
   } catch (err) {
-    throw new Error('Linter validation contains error(s)');
+    errorsFound = true;
+    console.error('Linter validation contains error(s)');
```

Comment:
print the error also ?

Reply:
The Linter itself will print the error. So, printing it again is not required.